### PR TITLE
Increase number of max connections to 8192

### PR DESCRIPTION
**Pull request by DockToFuture**
**How to categorize this PR?**

/area networking
/kind enhancement

**What this PR does / why we need it**:
This PR increases the number of max connections for envoy-proxy sidecar in vpn-seed-server to `8192`.
We saw on some shoot clusters that the default value of a maximum of 1024 connection was not sufficient for the envoy-proxy and led to a lot of 408 errors because the buffer of the envoy-proxy was overflowing.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
The number of max connections for envoy-proxy sidecar in vpn-seed-server is now increased to `8192`.
```

### CODE REVIEW

File: pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go [hunk 1]
```diff
@@ -819,6 +819,9 @@ var envoyConfig = `static_resources:
   clusters:
   - name: dynamic_forward_proxy_cluster
     connect_timeout: 20s
+    circuitBreakers:
+        thresholds:
+        - maxConnections: 8192
```
Comment (ialidzhikov) [inline pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go]: Shouldn't this be indented as

```suggestion
    circuitBreakers:
      thresholds:
      - maxConnections: 8192
```

?

Comment (ScheererJ) [reply pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go]: Agreed.

**Review by ScheererJ — CHANGES_REQUESTED** [review]
Please fix the indentation.

**Review by ScheererJ — APPROVED** [review]

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**